packages/render-fidelity-tools/test/README.md

```diff
@@ -116,3 +116,48 @@ Quick Look skybox in renderers that we wish to appear at least somewhat
 comparable to Quick Look: [quick_1k.png](../../examples/assets/quick_1k.png).
 
 
+## Adding CLI renderer backends
```

Comment:
Github won't let me add comments to binary files, so I'll just put this here. What's up with the extra labels on the MetalRoughSpheres render?

Reply:
Looks like missing backface culling. Unfortunately, there's nothing I can do about it right now. 
I can fix the missing backgroud though :) Still useful or too much confusion because of the labels?

Reply:
It's okay, we can ignore the extra labels for now; at least I understand them now.